web_src/js/features/projects.js

```diff
@@ -59,6 +59,21 @@ export default async function initProject() {
       });
   });
 
+  $(document).on('click', '.set-default-project-board', async function (e) {
+    e.preventDefault();
+
+    await $.ajax({
+      method: 'POST',
+      url: $(this).data('url'),
+      headers: {
+        'X-Csrf-Token': csrf,
+        'X-Remote': true,
+      },
+      contentType: 'application/json',
+    });
+
+    setTimeout(window.location.reload(true), 2000);
```

Comment:
```suggestion
    reload();
```

Is equivalent. Or ` setTimeout(() => window.location.reload(true), 2000);` if you want the timeout.